src/assets/_project/_blocks/components/forms/recaptcha.js

```diff
@@ -9,51 +9,114 @@
 import keys from '../../data/qg-google-keys';
 
 (function ($, swe) {
-  let googleRecaptchaApiKey =
-    window.location.hostname.search(
-      /\bdev\b|\btest\b|\blocalhost\b|\buat\b/
-    ) !== -1
-      ? keys.defGoogleRecaptcha.uat
-      : keys.defGoogleRecaptcha.prod;
-  let onloadRecaptcha = () => {
-    // eslint-disable-line
-    $('form[data-recaptcha="true"]')
-      .find('input[type="submit"], button[type="submit"]')
-      .on('click', e => {
-        e.preventDefault();
-        let subBtn = e.target;
-        let form = $(subBtn).parents('form');
-        try {
-          grecaptcha.render(subBtn, {
-            sitekey: googleRecaptchaApiKey, //this value will be replaced by build tool. from gulp-config/
-            callback: () => {
-              var response = grecaptcha.getResponse();
-              if (
-                response === '' ||
-                  response === undefined ||
-                  response.length === 0
-              ) {
-                console.log('Invalid recaptcha');
-                return false;
-              } else {
-                form.submit();
-              }
-            },
-          });
-        } catch (e) {
-          grecaptcha.reset();
+  let checkEnv = window.location.hostname.search(/\bdev\b|\btest\b|\blocalhost\b|\buat\b/);
+  let $feedbackForm = $('#qg-page-feedback-form');
+
+  let setUrlEnableCaptcha = () => {
```

Comment:
you may need to wrap this just in case the footer feedback does not exist.

Reply:
Not required but I have added here because if the form is not present it will evaluate to undefined and this will evaluate the else condition which is to do nothing and adding the attribute to an element not present does not throw an error.  

!$feedbackForm.attr('data-recaptcha') ? $feedbackForm.attr('data-recaptcha', 'true') : '';

$('#something-not-present').attr('data-recaptcha')
==> undefined